docs/source/tutorials/base/how_to_train/detection.rst

```diff
@@ -0,0 +1,382 @@
+Object Detection model
+======================
+
+This tutorial reveals end-to-end solution from installation to model export and optimization for object detection task on a certain example.
+On this page we show how to train, validate, export and optimize ATSS model on WGISD public dataset.
+
+.. note::
+
+  To learn how to deploy the trained model, refer to: :doc:`../deploy`.
+
+  To learn how to run the demo and visualize results, refer to: :doc:`../demo`.
+
+The process has been tested on the following configuration.
+
+- Ubuntu 20.04
+- NVIDIA GeForce RTX 3090
+- CUDA Toolkit 11.4 
+
+
+
+*************************
+Setup virtual environment
+*************************
+
+You can follow the installation process from a :doc:`quick_start guide <../../../get_started/quick_start>` to create a universal virtual environment for all tasks. On the other hand, you can create a task-specific environment following the process below.
+
+1. Check your python version:
+
+.. code-block::
+
+    python3 --version; pip3 --version; 
+
+    Python 3.8.10
+
+2. Create and activate a virtual environment for the obect detection task.
+The following example creates a virtual environment in the ``det_venv`` folder for detection task.
+
+.. code-block::
+
+    sudo apt-get install python3-venv
+    # create virtual env
+    bash ./otx/algorithms/detection/init_venv.sh det_venv
+    # activate virtual env
+    source det_venv/bin/activate
+
+.. #TODO: Update installation part: virtual env, install prerequisite, pip install -e . or pip install otx
+
+***************************
+Dataset preparation
+***************************
+
+1. Clone a repository with `WGISD dataset <https://github.com/thsant/wgisd>`_. 
+
+.. code-block::
+
+  cd data
+  git clone https://github.com/thsant/wgisd.git
+  cd wgisd
+  git checkout 6910edc5ae3aae8c20062941b1641821f0c30127
+
+
+This dataset contains images of grapevines with the annotation for different varieties of grapes. 
+
+- ``CDY``	- Chardonnay
+- ``CFR``	- Cabernet Franc
+- ``CSV``	- Cabernet Sauvignon
+- ``SVB``	- Sauvignon Blanc
+- ``SYH``	- Syrah
+
+It's a great example to start with. The model achieves high accuracy righ from the beginning of the training due to relatively large and focused objects. Also, these objects are distinguished by a person, so we can check inference results just by looking at images.
+
+.. image:: ../../../../../utils/images/wgisd_dataset_sample.jpg
```

Comment:
Need to be fixed
![image](https://user-images.githubusercontent.com/80735344/213085813-5dfec9ab-f4be-4df4-aace-58a1e7e80918.png)

Reply:
Sure